test/tests/functional/pbs_svr_dyn_res.py

```diff
@@ -88,3 +94,221 @@ def test_invalid_script_out(self):
         self.scheduler.log_match(
             "Insufficient amount of server resource: %s (R: 1 A: 0 T: 0)"
             % (resname))
+
+        self.du.rm(path=tmpfilepath, sudo=True)
+
+    def test_res_long_pos(self):
+        """
+        Test that server_dyn_res accepts command line arguments to the
+        commands it runs. Resource value set to a positive long int.
+        """
+        self.server.manager(MGR_CMD_SET, SERVER, {'scheduling': 'False'})
+
+        # Create a resource of type long
+        resname = "foobar"
+        attr = {"type": "long"}
+        self.server.manager(MGR_CMD_CREATE, RSC, attr, id=resname, expect=True)
+
+        # Add resource to sched_config's 'resources' line
+        self.scheduler.add_resource(resname)
+
+        # Add server_dyn_res entry in sched_config
+        self.scheduler.set_sched_config(
+            {'server_dyn_res': '"foobar !/bin/echo 4"'})
+
+        self.server.manager(MGR_CMD_SET, SERVER, {'scheduling': 'True'})
+
+        # Restart the server so the resourcedef info will be updated
+        self.server.restart()
+
+        # Setup node
+        a = {'resources_available.ncpus': 2}
+        self.server.manager(MGR_CMD_SET, NODE, a,
+                            id=self.mom.shortname, expect=True)
```

Comment:
i think there is no need to set ncpus value on node , in test only 1 job is submited without any no of ncpus.